README.md

```diff
@@ -52,7 +62,7 @@ Please download NetMHC using appropriate licensing. Download and installation in
 
```

Comment:
Right, we just say it's a prerequisite. They've gotta go to the NetMHC site and handle clicking through the license terms themselves, right?  Unless they have to do something special on that site, we don't really add information with a vague statement about licensing, right?